Tensile/KernelWriter.py

```diff
@@ -222,16 +234,24 @@ def makeSchedule(self, kernel, tensorParametersA, tensorParametersB, localWriteE
 
       itemsGRToSched.append(globalReadIncACode)
       for i in range(numEmptyGlobalReadIncCode):
-        itemsGRToSched.append(Code.Module())
+        imod = Code.Module()
+        imod.addCode(Code.Inst("// ", "globalReadIncA scheduler placeholder"))
+        itemsGRToSched.append(imod)
       itemsGRToSched.append(globalReadIncBCode)
       for i in range(numEmptyGlobalReadIncCode):
-        itemsGRToSched.append(Code.Module())
+        imod = Code.Module()
+        imod.addCode(Code.Inst("// ", "globalReadIncB scheduler placeholder"))
```

Comment:
can use addcomment0()

Reply:
The placeholder needs to be `Code.Inst` instead of `Code.TextBlock` so it can have an effect on scheduling decisions